Special abilities v2, mostly Flak (#4)
GameBrains/er-core: modules/special-rules/pages/infiltrators.adoc

= Infiltrators

These units are the expert scouts and commandos who gather information, set ambushes, sabotage installations, and so on.

---

> Apply these rules to {infiltrators} units:

Author: Should this be?
```suggestion
Apply these rules to units with the {infiltrators} ability:
```

Reviewer: I want to tweak this a bit further, along with the text below - but I'll apply this suggestion for now to resolve this conversation.

* Count them as *in command* while they are within 60cm of their HQ unit, rather than the usual 30cm.
However, they are still destroyed if they become Broken and they are more than 30cm away from their HQ unit.
* You may deploy them up to 30cm outside of your normal deployment zone for the scenario you are playing.

> Or, if you do not deploy them at the start of the battle, you may add +30cm to their first move later -- even if their first move is to exit from transport vehicles or drop pods.

Author: This should fix the issues in both TODO boxes.

```suggestion
Or, if they are not on the table at the start of the battle, you may add +30cm to their first move on the Turn they arrive -- even if their first move is to exit from transport vehicles or drop pods.
```

Reviewer: I don't like to obscure the agency of the player from the start of the sentence, or to break from the sense of the preceding sentence (which uses 'you' and 'deploy', not 'unknown actor' and 'on the table'). I might be overthinking the possibility that people might construe an inherent ability to **not** deploy.

I think the main point of your change is good — I just wouldn't capitalise 'turn' (I don't think we've done that anywhere else). I was wondering about 'turn in which', but thought of this instead, see what you think:
```suggestion
Or, if they are in reserve and so you do not deploy them at the start of the battle then, in the turn when they arrive, you may add +30cm to their first move -- even if their first move is to exit from transport vehicles or drop pods.
```
Bit of a long sentence, but you can't have it all. Other thoughts:

- If they're not deployed at the start, then they must be in reserve mustn't they? If this works then we could add Reserves to the related info block. 
- Possibly 'exit' should be 'disembark' -- depending on what terminology we want to use for transports. I went with embark/disembark for EA Rules Revamp — possibly because enter/exit didn't quite work in some cases, and because it works in hand with 'embarked/disembarked units'.

As an aside, I don't really like 'their first move' because it is the player that does the moving and I try to keep these things separate. But the alternative is probably a lot more wordy in this case, and I think this is an area where we (I) can live with this overlap.

Author: Internal design note:

I started by referencing Reserves, but had to remove it due to scenarios where detachments start off-table but **aren't** using the Reserves rule. In addition to Meeting Engagement, several of the refight scenarios use the same mechanic where you mark the arrival point of the detachment and they move on to the table in the first Movement phase, or allocate detachments to flank attacks and move them on in specific turns.

Exit and disembark - I agree that disembark is better.

I didn't spot this the first time through, but 'you may add +30cm to their first move' appears to contain an unneeded plus sign.

Author: This appears to be the combined suggestion, but I'm not happy about 'if you do not deploy them' as that infers that it's always the player's choice which isn't true as it is scenario-dependant.

```suggestion
Or, if you do not deploy them at the start of the battle then, in the turn when they arrive, you may add +30cm to their first move -- even if their first move is to disembark from transport vehicles or drop pods.
```

I know you don't like the change in actor, but I think this would be better:

```suggestion
Or, if they are not deployed at the start of the battle then, in the turn when they arrive, you may add +30cm to their first move -- even if their first move is to disembark from transport vehicles or drop pods.
```

> +
> +[TODO: Should we change this in some way to avoid the possible interpretation that "if you do not deploy them" could imply an inherent option rather than a possibility that depends upon the scenario setup rules?]+
> +[TODO: As written, it seems to me that you could interpret the rules to include the possibility that the infiltrators get their +30cm even if they disembark from a transport vehicle *in a later turn* than the turn they arrive in -- it'd still be their 'first move'. This can't also be intended and okay, can it?]+

Reviewer: ```suggestion
```
Not needed, once we settle the preceding text per the above conversation.

Author: Suggestion added to commit batch.
